modified scroll behaviour of review activity

vanshikaarora opened the pull request:
**Description (required)**
Fixes sub-task of #2698 [Scrolling behaviour of Review Activity]

### Changes made
Modified activity_review.xml and fragment_review_image.xml

**Screenshot**
![WhatsApp Image 2019-03-24 at 1 43 19 PM](https://user-images.githubusercontent.com/34261945/54876755-0d055100-4e3b-11e9-86e1-3b09fcf37d00.jpeg)

domdomegg commented on app/src/main/res/layout/activity_review.xml:
```diff
@@ -15,26 +15,29 @@
+                android:layout_height="25dp"
+                android:background="@android:color/transparent"
+                android:layout_below="@+id/toolbar"
+                android:text="SKIP THIS IMAGE"
```
I know it wasn't previously, but it should use a `@string` resource rather than hardcoded text.

vanshikaarora commented:
>I think it might be better if the whole page scrolled part from 'skip this image'. Currently just the bottom bit does.

@domdomegg before #2709 there was scrolling of whole activity instead of image, I removed whole activity scrolling in that PR .

In this current PR i adjusted the views to avoid scrolling every now and then. But it may scroll when occupies more than the extra space allotted. 

So should I revert the changes in scrolling that I did in #2709 ?

domdomegg commented:
Sorry, I mean the whole activity should scroll up and down, and only the questions should scroll sideways - if that makes sense.

vanshikaarora commented:
>I mean the whole activity should scroll up and down

Yes that's what happens previously. I'll revert the changes and send another PR :+1:

domdomegg commented:
Before your previous PR (f7e6b20cab7a5819a1181bf98261453b2675dcdb) this is what used to happen:

![untitled](https://user-images.githubusercontent.com/4953590/54878752-e9d1a600-4e28-11e9-91fc-822e4788b9b4.gif)

The whole activity scrolls up and down, and the whole activity scrolls sideways

After your PR, only the bottom bit scrolls up and down, and only the bottom bit scrolls sideways

I meant to change it so that the whole activity scrolls up and down, and only the bottom bit scrolls sideways.

vanshikaarora commented:
@domdomegg I have made the changes. Kindly review :)

vanshikaarora commented:
Thanks for approving the PR @domdomegg :)
